We don't include ncursesw.h header file, so use ncurses.h instead.

--- ./meson.build   2017-09-23 10:57:09.288877085 -0400
+++ ./meson.build   2017-09-23 10:58:40.263583716 -0400
@@ -83,6 +83,10 @@
     conf.set('HAVE_NCURSESW_CURSES_H', true)
   elif cc.has_header('ncursesw.h', dependencies: curses_dep)
     conf.set('HAVE_NCURSESW_H', true)
+  elif cc.has_header('ncurses/curses.h', dependencies: curses_dep)
+    conf.set('HAVE_NCURSES_CURSES_H', true)
+  elif cc.has_header('ncurses.h', dependencies: curses_dep)
+    conf.set('HAVE_NCURSES_H', true)
   else
     error('No ncursesw header found')
   endif
